### DISCUSSION TITLE
Cloudwatch Logs: Add default query in the query panel

### DISCUSSION THREAD

**Discussion by VermaPriyanka** [grafana/grafana#60735] [General]
### Customer Ask
Cloudwatch Logs Insights console today provides a default query as a "Getting Started" experience. Customers simply have to identify a Log group and can execute this query to explore the log data. They can further edit and enhance this query for their specific use-cases, but they get a starting point to start exploring the logs, without knowing the syntax.

### Proposal
The proposal is to define similar experience in the Explore panel and the Query editor for Cloudwatch logs datasource by pre-populating the generic query.

`“fields @timestamp, @message
| sort @timestamp desc
| limit 20`

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
